app/controllers/v0/sign_in_controller.rb

```diff
@@ -175,31 +180,36 @@ def revoke_all_sessions
 
     def logout # rubocop:disable Metrics/MethodLength
       client_id = params[:client_id].presence
-      anti_csrf_token = anti_csrf_token_param.presence
-
-      if client_config(client_id).blank?
+      if client_id && client_config(client_id).blank?
```

Comment:
Was there an issue just with `client_config(client_id).blank?`? It should still work even if `client_id` is `nil`, I don't think you need the extra check for client_id?